modules/web/src/main/java/org/jboss/weld/module/web/context/beanstore/http/AbstractSessionBeanStore.java

```diff
@@ -34,13 +37,16 @@
 
     protected abstract HttpSession getSession(boolean create);
 
+    private final ServiceRegistry serviceRegistry;
```

Comment:
`WeldConfiguration` is immutable so I'd rather store the boolean value of the config property.

Reply:
+1